app/subscriber/src/features/my-folders/ConfigureFolder.tsx

```diff
@@ -23,16 +23,16 @@ import {
 } from 'tno-core';
 
 import { getFilterOptions } from './constants';
+import { useMyFolderContext } from './MyFolderContext';
 import { Schedule } from './Schedule';
 import * as styled from './styled';
 import { createSchedule } from './utils';
 
-export interface IConfigureFolderProps {
-  active?: IFolderModel;
-}
+export interface IConfigureFolderProps {}
```

Comment:
I've a PR that fixes issues related to this.  While I like using contexts for shared components, I don't think that will fix the issues.

Reply:
But I'm all for it if you found it has fixed an issue.

Reply:
There was a place where we had another state variable that became out of sync and I replaced that with the context provider. Going to do some testing here though to see with the new changes

Reply:
I've marked it in a separate comment